native_client/swift/deepspeech-ios.podspec

```diff
@@ -0,0 +1,18 @@
+# Pull in version from outside
+version = File.read(File.join(__dir__, "../../training/deepspeech_training/VERSION")).split("\n")[0]
+
+Pod::Spec.new do |s|
+  s.name         = "deepspeech-ios"
+  s.version      = version
+  s.summary      = "DeepSpeech"
+  s.homepage     = "https://github.com/mozilla/DeepSpeech"
+  s.license      = "Mozilla Public License 2.0"
+  s.authors      = "Mozilla et al."
+
+  s.platforms    = { :ios => "13.5" }
```

Comment:
We declare `minimum_os_version=9.0` above, and here we push for 13.5 ?

Reply:
There are two frameworks and libraries at play now, right.

1. The static TensorFlow framework
2. The DeepSpeech swift lib that wraps (1)

For (1), I went for whatever minimum version I could. For (2), we have to go with a version that supports all the libs you all use for audio capture, etc. So that might be higher, but we can waddle our way back here and on the deepspeech_ios Xcode project until we can go no further.

Reply:
OK, I think on second look and test, 9.0 actually works here. At this stage, we're not using any audio libraries yet -- that comes in through the example app, so 9.0 should be good. (I was misspeaking, since I wrap the deepspeech source with my own audio libs in my own setup.)